docs/source/howto/workflows.rst

```diff
@@ -20,6 +20,159 @@ Running a predefined workflow
+
+.. code-block:: python
+
+    MultiplyAndAddWorkChain = WorkflowFactory('arithmetic.multiplyadd')
+
+The input argument of the ``WorkflowFactory`` is a string called the *entry point*.
+For each work chain, the entry point usually starts with with the name of the plugin package (``arithmetic``), followed by a dot and the string identifying the work chain (``multiplyadd``).
+Using the ``WorkflowFactory`` has the advantage that the entry point will not change when the packages or plugins are reorganised, and hence your code is less likely to break when updating AiiDA or the plugin that supplies the work chain.
+
+The list of installed plugins can be easily accessed via the verdi CLI:
+
+.. code-block:: console
+
+    $ verdi plugin list
+
+To see the list of ``WorkChain`` entry points, simply use:
+
+.. code-block:: console
+
+    $ verdi plugin list aiida.workflows
+
+By further specifying the entry point of the work chain, you can see its description, inputs, outputs and exit codes:
+
+.. code-block:: console
+
+    $ verdi plugin list aiida.workflows arithmetic.multiplyadd
+
+.. note::
+
+    The list of available plugins for AiiDA can be found at the `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry/>`_.
+    If you want to write your own workflow, please see the section on :ref:`how to write your own workflows<how-to:workflows:write>`.
+
+Work functions
+--------------
+
+Running a work function is as simple as calling a typical Python function, simply call the function with the required input arguments:
+
+.. code-block:: python
+
+    from aiida.workflows.plugins.arithmetic.addmultiply import add_and_multiply
+    Int = DataFactory('int')
+
+    result = add_and_multiply(Int(2), Int(3), Int(5))
+
+Here, the ``add_and_multiply`` work function returns the output ``Int`` node and stores it in the variable ``result``.
+Note that the input arguments of a work function must be a subclass of the ``Data`` class.
+Just calling the ``add_and_multiply`` function with regular integers will result in a ``ValueError``, as these cannot be stored in the provenance.
+
+.. note::
+
+    Although the example above shows the most straightforward way to run the ``add_and_multiply`` work function, there are several other ways of running processes that can return more than just the result.
+    For example, the ``run_get_node`` function from the AiiDA engine returns both the result of the workflow and the work function node.
+    See the :ref:`corresponding topics section for more details <topics:processes:usage:launching>`.
+
+Work chains
+-----------
+
+To launch a work chain, you can either use the ``run`` or ``submit`` functions from the AiiDA engine.
+For either function, you need to provide the class of the work chain as the first argument, followed by the inputs as keyword arguments.
+Running a work chain means it is executed in the same interpreter in which it is launched:
```

Comment:
I'd even suggest to be more explicit: "in the same process as the interpreter in which it is launched".

Reply:
Fine with being more explicit. We should just be careful with the ambiguity of the word "process" in these contexts. You mean the system process in this case, but in a context of AiiDA it may easily be misunderstood for the `Process` class.

Reply:
Maybe explicitly say it is a system process?

"Using the `run` function, or "running", a work chain means it is executed in the same system process as the interpreter in which it is launched"